ext/node/polyfills/internal/buffer.mjs

```diff
@@ -91,20 +91,20 @@ import {
   forgivingBase64UrlEncode,
 } from "ext:deno_web/00_infra.js";
 import { atob, btoa } from "ext:deno_web/05_base64.js";
-import { Blob } from "ext:deno_web/09_file.js";
+import { Blob, File } from "ext:deno_web/09_file.js";
 
-export { atob, Blob, btoa };
+export { atob, Blob, File, btoa };
```

Comment:
Seems like this should be in a separate PR? Also, could you revert the submodule changes?